src/pages/FuFit.jsx

```diff
@@ -0,0 +1,184 @@
+                                <hr className="w-16 h-1  bg-[#FFA523] border-0 rounded mt-[5%]"/>
+                                <p className="text-xs text-black mt-[5%] leading-[22px]">Our recreational exercise program provides general members 
+                                an outlet to stay active and healthy. Participate in a variety of activities designed to suit all fitness levels, 
+                                including basketball, volleyball, soccer, flag football, ultimate frisbee, pickleball, kickball, yoga, group workouts, 
+                                and many more. Whether you are looking to work on your fitness goals or simply want to stay active, FUSION Fits have 
+                                something to offer for everyone at all levels. Join us to stay active, connect with fellow members, and make fitness 
+                                an enjoyable part of your routine!</p>
+                            </div>
+                        </div>
+                        <img src={fufit_shadow1} className="w-64 h-8 mt-5 mx-auto"></img>
+                    </div>
+                </div>
+            </section>
+
+            {/* bring the heat */}
+            <section className="w-full h-[540px] bg-[#F4F4F4] flex flex-row relative">
+                <div className="absolute inset-0 z-5">
+                    <img src={fufit_bg2} className="w-full h-[540px] object-cover"></img>
+                </div>
+
+                <div className="w-[53.5%] h-full relative z-10">
+                    <div className="w-[63%] h-full flex flex-col ml-[22%]">
+                        <div className="w-full h-[76%] bg-white mt-10 rounded-b-[60px] rounded-tl-[60px]">
+                            <div className="w-[85%] ml-[7%]">
+                                <h1 className="font-[700] text-[#FAC80A] mt-[7%] tracking-[.1em]">FUSION FIT &gt;&gt;</h1>
+                                <h1 className="text-black text-3xl font-[700] mt-[3%] ">Bringing the Heat to Every Game</h1>
+                                <hr className="w-16 h-1  bg-[#FAC80A] border-0 rounded mt-[5%]"/>
+                                <p className="text-xs text-black mt-[5%] leading-[22px]">Our recreational exercise program provides general 
+                                members an outlet to stay active and healthy. Participate in a variety of activities designed to suit all 
+                                fitness levels, including basketball, volleyball, soccer, flag football, ultimate frisbee, pickleball, kickball, 
+                                yoga, group workouts, and many more. Whether you are looking to work on your fitness goals or simply want to 
+                                stay active, FUSION Fits have something to offer for everyone at all levels. Join us to stay active, connect 
+                                with fellow members, and make fitness an enjoyable part of your routine!</p>
```

Comment:
Same issue above, with the spacing, I would also to make this easier for yourself make a component since you're basically using the same thing three times